### PR TITLE
add docs job

### LINKED ISSUE #1500: Docs updation job is absent/broken.
https://maven.fabric8.io/ 

Docs were last updated for Version 3.5.41,
date 2018-10-16 . 

So evidently, the ci-docs job isn't running.

### PR DESCRIPTION
**Pull request by dev-gaur**
fixes #1500

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -119,10 +116,10 @@ workflows:
   version: 2
   all:
     jobs:
-    - doc:
-        filters:
-          branches:
-            only: 3.5.x
+    - doc
+#        filters:
```
Comment (rohanKanojia) [inline .circleci/config.yml]: remove these comments.

File: samples/yaml-only/src/main/fabric8/deployment.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2016 Red Hat, Inc.
+#
+# Red Hat licenses this file to you under the Apache License, version
```
Comment (rohanKanojia) [inline samples/yaml-only/src/main/fabric8/deployment.yml]: Seems like you add these files by accident

File: threads_report.txt [hunk 3]
```diff
@@ -0,0 +1,63 @@
+====
+    Copyright 2016 Red Hat, Inc.
+
+    Red Hat licenses this file to you under the Apache License, version
+    2.0 (the "License"); you may not use this file except in compliance
```
Comment (rohanKanojia) [inline threads_report.txt]: same here

File: .circleci/config.yml [hunk 4]
```diff
@@ -119,10 +116,10 @@ workflows:
   version: 2
   all:
     jobs:
-    - doc:
+    - doc
```
Comment (rohanKanojia) [inline .circleci/config.yml]: You missed a trailing colon here: `doc:`

Comment (dev-gaur) [reply .circleci/config.yml]: oops

**Pull request merged**